src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexParser.cs

```diff
@@ -2141,22 +2093,6 @@ private void AddAlternate()
             _concatenation = new RegexNode(RegexNodeKind.Concatenate, _options);
         }
 
-        /// <summary>Finish the current quantifiable (when a quantifier is not found or is not possible)</summary>
-        private void AddConcatenate()
-        {
-            // The first (| inside a Testgroup group goes directly to the group
-
-            _concatenation!.AddChild(_unit!);
-            _unit = null;
-        }
-
-        /// <summary>Finish the current quantifiable (when a quantifier is found)</summary>
-        private void AddConcatenate(bool lazy, int min, int max)
-        {
-            _concatenation!.AddChild(_unit!.MakeQuantifier(lazy, min, max));
-            _unit = null;
-        }
```

Comment:
NIT: I would keep this methods as is, they have 2 statements and has some comment could be useful for readers, but up to you.

Reply:
Thanks, but I find it clearer